app/javascript/src/controllers/plan_controller.js

```diff
@@ -1,4 +1,5 @@
 import { Controller } from "stimulus"
+import $ from "jquery"
```

Comment:
Why is this necessary?

Reply:
jest complains that `$` is undefined in the controller since its unaware of the webpacker environment.